### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59496]
#### VFS team name
MHV

#### VFS product name
My HealtheVet

#### VFS feature name
Secure Messaging

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
On [/secure-messages/compose](https://staging.va.gov/my-health/secure-messages/compose) (after the interstitial), the "Start a new message" H1 is followed by a "New message" H3 with no intermediate H2. The heading hierarchy is used by screen reader users for navigation and wayfinding, and skipped heading levels can cause confusion.

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

### Platform Recommendation
"New message" should be an H2.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by lichellebain** [comment]
Jira ticket created:  https://jira.devops.va.gov/browse/MHV-46059

**Comment by vsaleem** [comment]
Good morning! @shiragoodman @AngelaFowler82 @sara-amanda -- This issue has been implemented by our team and is now in the staging environment.  Can we please have some help with this accessibility testing to see if this is an issue?
